### PR TITLE
infographic fixes:

### PR DESCRIPTION
**Pull request by lucernae**
### What does it fix ?
* Ticket #
* Funded by : DFAT
* Description : 
- Change fallback font to Arial in Windows
- Create fallback header if affected field not found

### CODE REVIEW

File: resources/css/InaSAFE.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 body {
-  font-family: 'Ubuntu', 'Lucida Grande', 'Segoe UI', sans-serif;
+  font-family: 'Ubuntu', 'Lucida Grande', 'Segoe UI', 'Arial', sans-serif;
```
Comment (timlinux) [inline resources/css/InaSAFE.css]: Maybe you should give hight priority to Arial? Can you get someone to confirm if this fixes it?

**Comment by timlinux** [comment]
@lucernae can we get this merged?

**Comment by lucernae** [comment]
I haven't tested it on Windows yet. I haven't set up the environment. So, no screenshot for now.

**Comment by lucernae** [comment]
I tested it on Windows. It seems QGIS Composer in Windows doesn't change the font in HTML Frame at all. I even tried to use User Stylesheet in Composer but it's not effective.
So, I will merge it for now.

**Comment by lucernae** [comment]
I created issue #4058 For this CSS problem.
For this PR, no layout/presentation changes happens.

**Pull request merged**